### DISCUSSION TITLE
Noah-MP 3.6 basics

### DISCUSSION THREAD

**Discussion by sanjaync** [NASA-LIS/LISF#1587] [Q&A] [answered]
Hello all,

I just finished running the website test case. I want to start working with the Noah-MP 3.6 model with GDAS data for the Australian region. Can anyone suggest where I should begin? I have gone through the documentation, but I am still confused. Where can I find example configuration files? Please help me out.

**Comment by dmocko** [comment, marked as answer]
Hi @sanjaync 

Congrats at successfully going through the testcase!

We try not to distribute example config files, as we would need to update all of them every time we update the code.

My best recommendation is to look at the LIS Users' Guide:
https://nasa-lis.github.io/LISF/LIS_users_guide/LIS_users_guide.html

Look for the Noah-MP.3.6 and GDAS example lis.config entries, including the different and recommended options.

**Comment by dmocko** [comment]
One other reminder.  If you are using Noah-MP (3.6 or 4.0.1), you need to be sure you have this config entry in your LDT config:
![Screenshot 2024-08-09 at 11 19 58 AM](https://github.com/user-attachments/assets/c3eea0cf-dfe7-4998-8430-65311f8def87)
